File: src/leetcode/easy/Solution26.java
package leetcode.easy;

import java.util.Arrays;

public class Solution26 {
    int count =0;
    int j=1;
    public int removeDuplicates(int[] nums) {

        if(nums.length==0){
            return count;
        }else{
            count=1;
        }

        for(int i=1; i<nums.length; i++){
            if(nums[i-1] != nums[i]){
                nums[j] = nums[i];
                count++;
                j++;
            }
        }

        Arrays.copyOf(nums, j);

        return count;
    }
}

File: src/leetcode/easy/Solution26.cs
// https://leetcode.com/problems/remove-duplicates-from-sorted-array/description/

public class Solution26 {
    public int RemoveDuplicates(int[] nums) {
        int index =0;
        int uniqueIndex = 0;

        while(index <= nums.Length-1)
        {
            if(nums[uniqueIndex] == nums[index])
            {
                index++;
            }
            else
            {
                uniqueIndex++;
                nums[uniqueIndex] = nums[index];
            }
            
        }
        
        return uniqueIndex+1;

    }
}
